dist_filter_max: 0.4
dist_filter_min: 0.02
ransac_trials: 50
ransac_eps_thresh: 0.15
ransac_ctrlset_size: 180
ransac_phi_max: 45.0
registration_mode: 3
use_odom_rescue: false
icp_iterations: 30
reg_trs_max: 1.0
reg_sin_rot_max: 0.5
trials: 30
epsTresh: 0.15
sizeControlSet: 360
zrand: 0.05
zhit: 0.2
zphi: 0.2
zshort: 0.2
zmax: 0.2
percentagePointsInC: 0.6
rangemax: 30.0
sigphi: 0.052
sighit: 0.1
lamshort: 0.08
maxAngleDiff: 1.5
maxAnglePenalty: 0.5
las_min_range: 0.26
min_range: 0.001
max_range: 30.0
truncation_radius: 10
use_object_inflation: true
object_inflation_factor: 1